plugins/aws/sts_provisioner.go

```diff
@@ -231,11 +243,7 @@ func resolveLocalAnd1PasswordConfigurations(itemFields map[sdk.FieldName]string,
 	}
 
 	if awsConfig.HasMfaSerial() && awsConfig.MfaToken == "" {
-		return fmt.Errorf("MFA failed: an MFA serial was found but no OTP has been set up in 1Password")
-	}
-
-	if !awsConfig.HasMfaSerial() && awsConfig.MfaToken != "" {
-		return fmt.Errorf("MFA failed: an OTP was found wihtout a corresponding MFA serial")
+		return fmt.Errorf("MFA failed: an MFA serial (%s) was detected on the associated item for the selected profile, but no 'One-Time Password' field was found. Add an OTP to your item to use multi-factor authentication", awsConfig.MfaSerial)
```

Comment:
Let's rephrase this to something more readable:
```suggestion
		return fmt.Errorf("MFA failed: MFA serial %q was detected on the associated item for the selected profile, but no 'One-Time Password' field was found. Add an OTP to your item to use multi-factor authentication", awsConfig.MfaSerial)
```

Reply:
Also, this may be a good place to link a docs link to how to add an OTP to the item, it's not something self explanatory.

Reply:
Do we generally add links to error messages? It would make this specific error message rather long:

```
MFA failed: an MFA serial (%s) was detected on the associated item for the selected profile, but no 'One-Time Password' field was found. Learn how to add an OTP field to your item: https://developer.1password.com/docs/cli/shell-plugins/aws/#optional-set-up-multi-factor-authentication

```
(as per Lucy's suggestion)

Reply:
I'd definitely add some spacing in there, for better readability:
```
MFA failed: an MFA serial (%s) was detected on the associated item for the selected profile, but no 'One-Time Password' field was found. 

Learn how to add an OTP field to your item: 

https://developer.1password.com/docs/cli/shell-plugins/aws/#optional-set-up-multi-factor-authentication
```

Reply:
> Do we generally add links to error messages?

not consistently, although I believe we should do this. Not providing an actionable way for the user to proceed, after getting an error message, is very frustrating and can be a bottleneck for retention

Reply:
Okay, added it. Done!